docs/immortal-empires/norsca.md

```diff
@@ -0,0 +1,30 @@
+# Norsca
+
+## Wulfrik the Wanderer - World Walkers
+
+* **Route I - Ragnarok**
+  * The End Times are nigh. Wulfrik has long cursed the gods for his predicament, but now realises he must pledge himself to one of them if he is to survive the coming apocalypse.
+  * Attain level 3 allegiance with a Chaos God.
+   * Defeat all of the Challengers sent against you.
+  * **Reward**:

```

Comment:
Given that this objective relates to chaos gods I wonder if something with daemons/corruption/chaos as part of the reward would make sense?

Reply:
Could unlock some daemons for him but it would probably be very difficult for you guys because you would need to detect which god the player allied with. So if he allied with the Hound you could give him ability to recruit Khorne bloodletters etc.

Otherwise could just whack some diplomacy/allegiance point buffs for him but again would have to detect which god he pledged to unless you make it just like an undivided daemon thing which wouldn't make a super lot of sense but would be the easiest way.

Keep in mind getting lvl 3 gives you a powerful reward anyway, we're really just putting a reward on top of a reward here with this objective so we shouldn't go too hard.

Reply:
> Could unlock some daemons for him but it would probably be very difficult for you guys because you would need to detect which god the player allied with. So if he allied with the Hound you could give him ability to recruit Khorne bloodletters etc.

So this isn't that complex - they're four separate resources:

nor_progress_crow	        0	100
nor_progress_eagle	        0	100
nor_progress_hound	0	100
nor_progress_serpent	0	100

The scripted objective can check, each turn, if any are level 3 and run that unlock function, then set a flag to prevent further unlocks (or, if we're feeling generous, let them unlock more god's units as each god reaches level 3). I think this would be a really useful extension of his unique mechanic for a route.

Reply:
Need to decide which units we unlock then because there's a lot to choose from. Daemon only or mortal only or both?

Reply:
This is tricky. I would personally prefer mortal only, but all mortal chaos stuff is locked behind a paywall. I would go with the classical four everyone is going to have: Bloodletters, Daemonettes, Plaguebearers and Horrors.

Reply:
I was leaning towards demons, but for a different reason: Norsca's units are all mortals, let's not give them more mortals.

Reply:
I'd suggest we keep this as unlocking demonic units, the scripting is minimal and can be mostly be copied from other scripts. I assume they can unlock only one god's unit - the first one they reach level 3 with a god?

Reply:
I agree, sounds cooler overall.

Reply:
Basic + exalted? E.g. bloodletters + exalted bloodletters?

Reply:
Sounds good.

Reply:
Have added this to docs, will be in next commit.